doc/api/perf_hooks.md

```diff
@@ -198,6 +210,31 @@ and can be queried with `performance.getEntries`,
 observation is performed, the entries should be cleared from the global
 Performance Timeline manually with `performance.clearMarks`.
 
+### \`performance.markResourceTiming(timingInfo, requestedUrl, initiatorType,
```

Comment:
Maybe add a statement like "This property is an extension by Node.js. It is not available in Web browsers", like what `performance.timerify` does?

Reply:
That's fair. Fixed.